app/views/categories/show.html.erb

```diff
@@ -16,11 +16,4 @@
             <p>$<%= expense.amount %></p>
         </div>
     <% end %>
-
-    <div class="footer fixed-bottom py-4">        
-        <%= link_to "Add Expense", new_expense_path, class: "btn main-color" %>        
-        <% if user_signed_in? %> 
-           <%= link_to "Logout", destroy_user_session_path, data: { turbo_method: :delete }, class: "btn main-color" %>
-        <% end %>
-    </div>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/222922361-acf66d26-9d0f-47fd-8009-3f7b0eff7a52.png)

- [ ] It was great that you are following the reviewer requirements but I guess you break something here and because of that one of your tests is failing which you can see in the image above :arrow_up: Kindly check it and fix it because the test tells you if your application is behaving correctly or not :+1: